requirements-dev.txt

```diff
@@ -1,6 +1,7 @@
 -r requirements.txt
 setuptools>=56.0.0
 pytest==7.1.1
+nbmake
```

Comment:
no version ?

Reply:
it's not needed so no

Reply:
it is a package to transforme notebook to executable